Makefile

```diff
@@ -109,6 +110,14 @@ PLUGIN_PACKAGES += mattermost-plugin-nps-v1.1.0
 PLUGIN_PACKAGES += mattermost-plugin-welcomebot-v1.2.0
 PLUGIN_PACKAGES += mattermost-plugin-zoom-v1.5.0
 
+ifneq (Darwin,$(PLATFORM))
+	DOCKER_MAKEFILE := ./docker-compose.makefile.yml
+else ifeq (arm64, $(ARCH))
+    DOCKER_MAKEFILE := ./docker-compose.makefile.m1.yml
+else
+	DOCKER_MAKEFILE := ./docker-compose.makefile.yml
+endif
```

Comment:
Nit: spacing is not uniform.

Reply:
weird seems one is a tab and the other are spaces, probably something wrong in my config, will fix it